### PR TITLE
Jampad Submission

### PR DESCRIPTION
**Pull request by jamdotjar**
Ready to make a PR? Great! Make sure your directory structure looks like this:

├── CAD \
│   ├── case.stl \
│   └── plate.dxf \
├── firmware \
│   └── rp2040.bin \
└── PCB \
    ├── macropad.pro \
    └── macropad.sch

### CODE REVIEW

**Comment by M0HID** [comment]
Hi! Great work, however there is one .DXF file missing which is needed for the laser cutter to cut the plate. Could you add that to the PR please?

**Comment by jamdotjar** [comment]
Sorry, just for clarification which file is missing? I already have jampad-plate.dxf, so what would the other file be.

**Comment by jamdotjar** [comment]
> Hi! Great work, however there is one .DXF file missing which is needed for the laser cutter to cut the plate. Could you add that to the PR please?

Sorry, just for clarification which file is missing? I already have jampad-plate.dxf, so what would the other file be

File: hackpads/jampad/cad/case/Jampad-plate.dxf
Comment (jamdotjar) [inline hackpads/jampad/cad/case/Jampad-plate.dxf:1]: This is the dxf for lasercutting

**Comment by M0HID** [comment]
Yep, looks good now! You can order your Hackpad [here](https://forms.hackclub.com/hackpad)!

**Pull request merged**